inference-engine/src/vpu/graph_transformer/src/stages/interp.cpp

```diff
@@ -9,6 +9,16 @@
 #include <memory>
 #include <set>
 
+const char* coordTrans           = "coordinate_transformation_mode";
+const char* modeI                = "mode";
```

Comment:
I suggest you to use exactly the same name for variables as for values

Reply:
done, except for `type`. Explained below.